vassal-app/src/main/java/VASSAL/tools/image/ImageUtils.java

```diff
@@ -463,4 +467,60 @@ public static String stripImageSuffix(String name) {
     }
     return name;
   }
+
+  /**
+   * Create a viewable representation of a null or empty image to
+   * use as a place holder in Configurers.
+   *
+   * The image will contain the translated text for the key
+   * Editor.ImageUtils.no_image
+   *
+   * @return Viewable null image
+   */
+  public static Image createViewableNullImage() {
+    return createViewableNullImage(64, 64);
+  }
+
+  /**
+   * Create a viewable representation of a null or empty image to
+   * use as a place holder in Configurers.
+   * *
+   * The image will contain the translated text for the key
+   * Editor.ImageUtils.no_image
+   *
+   * @param minWidth Minimum width for generated the image
+   * @param height   Height of the generated image
+   * @return Viewable null image
+   */
+  public static Image createViewableNullImage(int minWidth, int height) {
+
+    final int FONT_SIZE = 12;
+    final Font FONT = new Font(Font.DIALOG, Font.ITALIC, FONT_SIZE);
+
+    // Determine the size of the translated string
+    final String s = Resources.getString("Editor.ImageUtils.no_image");
+    Image image = new BufferedImage(minWidth, height, BufferedImage.TYPE_4BYTE_ABGR);
+    Graphics2D g2d = (Graphics2D) image.getGraphics();
+    final double os_scale = g2d.getDeviceConfiguration().getDefaultTransform().getScaleX();
```

Comment:
Oh, just noticed this---this won't get the screen scale factor, because the Graphics2D isn't coming from the screen. I know what to do here, but it will take a small bit of refactoring to make it work, so you can leave this to me.

Reply:
Ah OK, thanks.

Reply:
PR here: https://github.com/BrentEaston/vassal/pull/10

Reply:
Done